generic_processing.conf

```diff
@@ -39,20 +39,16 @@
 [FILTER]
    Name modify
    Match  kube.*
-   Rename apm-qa/projectname _tag_projectName
    Rename snappyflow/component _tag_componentName
-   Rename apm-qa/appname _tag_appName
    Rename container_name _tag_containerName
 
 [FILTER]
-   Name   rewrite_tag
-   Match  kube.*
-   Rule   $_tag_componentName  nginx  $_tag_componentName  false
+   Name record_modifier
+   Match kube.*
+   Record _tag_projectName ${SFAPM_PROJECTNAME}
```

Comment:
There is no env variable SF_PROJECTNAME
what I said was, we will add env for label SF_PROJECTNAME_LABEL